### DISCUSSION TITLE
Custom navbar width, length and text size

### DISCUSSION THREAD

**Discussion by ghost** [quarto-dev/quarto-cli#5489] [Q&A] [answered]
Is there anything in the docs about customizing the navbar width, length and text size? If not, how could this be accomplished?

I have searched the documentation and I have found how to custom the sidebar width but not the navbar.

**Comment by mcanouil** [comment, marked as answer]
As for many things in HTML, these are controlled by (S)CSS.
Use your favourite browser developer mode to see and edit the proper CSS element and create a custom CSS.
